container/Dockerfile

```diff
@@ -28,8 +28,8 @@ RUN wget -O /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/j
 RUN chmod +x /usr/local/bin/jq
 
 # Install screen
-RUN wget -P /tmp/ http://mirror.centos.org/centos/7/os/x86_64/Packages/screen-4.1.0-0.25.20120314git3c2946.el7.x86_64.rpm
-RUN yum localinstall -y /tmp/screen-4.1.0-0.25.20120314git3c2946.el7.x86_64.rpm && yum clean all && rm -rf /var/cache/yum && rm -rf /tmp/screen-4.1.0-0.25.20120314git3c2946.el7.x86_64.rpm
+RUN wget -P /tmp/ http://mirror.centos.org/centos/7/os/x86_64/Packages/screen-4.1.0-0.26.20120314git3c2946.el7.x86_64.rpm
```

Comment:
Just curious why the change for screen ?

btw, if you're wondering, I added this, err, it was for the POT so users can run screen within the container for the sample kafka app that requires you to start a server (kafka broker) in the background.

Reply:
that is the new screen file, the old one is not there, talked to Christian about it.